03-Bash/fix_owner/kernel_hm.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
 
-# For every user in system with id >= 1000 check home folder.
+ask_user_yes_no() {
+
+  read -p "Would you like to change owner? y/n " -n 1 -r
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]
+  then
+    return 0
+  fi
```

Comment:
Can You use one line instead of these 5 lines?
Like [[ $REPLY =~ ^[Yy]$ ]]

Reply:
in this case error appears 'numeric is required'